FrejaEidClient/src/main/java/com/verisec/frejaeid/client/client/impl/AuthenticationClient.java

```diff
@@ -49,55 +53,72 @@ public static Builder create(SslSettings sslSettings, FrejaEnvironment frejaEnvi
+        LOG.debug("Received authetnication transaction reference {}.", reference);
+        return reference;
     }
-
+    
     @Override
     public AuthenticationResult getResult(AuthenticationResultRequest authenticationResultRequest) throws FrejaEidClientInternalException, FrejaEidException {
         requestValidationService.validateResultRequest(authenticationResultRequest);
-        return authenticationService.getResult(authenticationResultRequest);
+        LOG.debug("Getting result for authentication transaction reference {}.", authenticationResultRequest.getAuthRef());
+        AuthenticationResult authenticationResult = authenticationService.getResult(authenticationResultRequest);
+        LOG.debug("Received {} status for authentication trasnsaction reference {}.", authenticationResult.getStatus(), authenticationResult.getAuthRef());
+        return authenticationResult;
     }
-
+    
     @Override
     public List<AuthenticationResult> getResults(AuthenticationResultsRequest authenticationResultsRequest) throws FrejaEidClientInternalException, FrejaEidException {
         requestValidationService.validateResultsRequest(authenticationResultsRequest);
-        return authenticationService.getResults(authenticationResultsRequest).getAuthenticationResults();
+        LOG.debug("Getting all authentication transaction results.");
+        List<AuthenticationResult> authenticationResults = authenticationService.getResults(authenticationResultsRequest).getAuthenticationResults();
+        LOG.debug("Successfully received authentication results.");
+        return authenticationResults;
     }
-
+    
     @Override
     public AuthenticationResult pollForResult(AuthenticationResultRequest authenticationResultRequest, int maxWaitingTimeInSec) throws FrejaEidClientInternalException, FrejaEidException, FrejaEidClientPollingException {
         requestValidationService.validateResultRequest(authenticationResultRequest);
-        return authenticationService.pollForResult(authenticationResultRequest, maxWaitingTimeInSec);
+        LOG.debug("Polling {} s for result for authentication transaction reference {}.", maxWaitingTimeInSec, authenticationResultRequest.getAuthRef());
+        AuthenticationResult authenticationResult = authenticationService.pollForResult(authenticationResultRequest, maxWaitingTimeInSec);
+        LOG.debug("Received {} status for authentication trasnsaction reference {}, after polling for result.", authenticationResult.getStatus(), authenticationResult.getAuthRef());
+        return authenticationResult;
     }
-
+    
     @Override
     public void cancel(CancelAuthenticationRequest cancelAuthenticationRequest) throws FrejaEidClientInternalException, FrejaEidException {
         requestValidationService.validateCancelRequest(cancelAuthenticationRequest);
+        LOG.debug("Canceling authentication transaction with reference {}.", cancelAuthenticationRequest.getAuthRef());
```

Comment:
Canceled or cancelled is the past tense of the verb to cancel. Both spellings are correct; Americans favor canceled (one L), while cancelled (two Ls) is preferred in British English and other dialects. 
Now, I believe we tend to use British English dialect ...

Reply:
Ok 👍